roles/kiwix/tasks/install.yml

```diff
@@ -100,7 +100,7 @@
     - { src: 'iiab-make-kiwix-lib', dest: '/usr/bin/iiab-make-kiwix-lib', mode: '0755' }
     - { src: 'iiab-make-kiwix-lib3.py', dest: '/usr/bin/iiab-make-kiwix-lib.py', mode: '0755' }
 
-- name: daemon_reload, so systemd (re)reads kiwix-serve.service
+- name: daemon_reload, so systemd (re)reads newly installed kiwix-serve.service
```

Comment:
If you are aiming the comment at beginners 're(reads)' is not accurate, 'recognize' is. The finer point of systemd is newly installed service files won't be utilized until the service files are read the first time with `systemctl daemon-reload` as the newly installed service files did not exist when the machine was booted, so the service file were never read to begin with.

Reply:
The word 'recognize' is very wishy-washy and doesn't tell me much all :/

Personally, I find this far more useful:

> reloading the systemd configurations of unit files, rerun all generators and recreates the dependency tree.
>
> I guess since the dependency tree is regenerated it doesn't make sense to run this on a single unit as several unit files may have changed and the dependency tree then could be wrong."

https://serverfault.com/questions/980550/can-i-reload-a-single-systemd-unit-service-file-after-its-modification-without-r/980551#980551